cli/flox/src/commands/edit.rs

```diff
@@ -245,6 +244,11 @@ impl Edit {
             },
             EditResult::Success { .. } => message::updated("Environment successfully updated."),
         }
+
+        if result != EditResult::Unchanged {
+            warn_manifest_changes_for_services(flox, environment);
+        }
+
```

Comment:
nonblocking: I thought we decided we wanted to combine this with the other warning for edit? https://github.com/flox/flox/issues/1885#issuecomment-2286464730

Reply:
I overlooked the suggestion once I'd got into the implementation. I can look into it in a follow-up PR, but will require some refactoring because `edit` will no longer be able to use `warn_manifest_changes_for_services`.